lib/spack/spack/test/packages.py

```diff
@@ -77,30 +75,20 @@ def test_package_class_names(self):
     #
 
     def test_import_package(self):
-        import spack.pkg.builtin.mock.mpich
+        pass
```

Comment:
what happened with these test functions?  Looks like they're disabled in your branch.